src/redux/cart/cart.operations.js

```diff
@@ -107,6 +107,10 @@ const getCartByUserId = async (userId) => {
   `;
   const result = await getItems(getCartByUserIdQuery, { id: userId });
 
+  if (result?.data?.getCartByUserId?.message) {
```

Comment:
You don't need this.
getItes already throw the error with the message.
It should be handled in sagas error handler.

Reply:
fixed